app/Services/User/DeleteUserService.php

```diff
@@ -19,11 +19,10 @@ public function __construct(UserRepositoryInterface $userRepository)
     public function handle()
     {
         try {
-            $this->userRepository->delete($this->data->id);
-            return true;
+            return $this->userRepository->delete($this->data->id);
```

Comment:
Đối với những data quan trọng (thông tin user, thông tin thanh toán, ....) khi thực hiện delete thì nên sử dụng Soft Delete, em research thêm phần này nhé